src/containers/formContainer.js

```diff
@@ -3,6 +3,7 @@ import ManualEntryForm from '../components/form/form';
 
 const FormContainer = reduxForm({
     form: "Manual_Entry",
+    touchOnChange: true,

```

Comment:
For some reason, the animations on this form are jarring when adding or removing characters. Its as if they change their mind halfway, and then decide to go ahead anyway (so when adding a character, it kinda appears, disappears, and then appears). Not sure if that's my phone or android thing.